src/Aspire.Cli/Commands/RunCommand.cs

```diff
@@ -64,6 +64,9 @@ public RunCommand(
         _serviceProvider = serviceProvider;
         _sdkInstaller = sdkInstaller;
 
+        var projectArgument = new Argument<FileInfo?>("project");
+        Arguments.Add(projectArgument);
```

Comment:
Why did you need this?

Reply:
This is just so we can `aspire run apphost.cs` instead of `aspire run --project apphost.cs`. Eventually I'll get the scanning working - but I didn't touch that for this pass.